src/operator/contrib/nnvm_to_onnx.cc

```diff
@@ -78,8 +84,13 @@ op::TRTParam ConvertNnvmGraphToOnnx(
 
   ModelProto model_proto;
   // Need to determine IR versions and features to support
-  model_proto.set_ir_version(static_cast<int64>(2));
+  auto opset_proto = model_proto.add_opset_import();
+  opset_proto->set_version(static_cast<int64>(8));
```

Comment:
Magic value? Is there some dynamic approach to retrieve them (e.g. from tensor rt)  or can we extract them into some documented constants?

Reply:
Yeah I could refactor these to give stronger names.  It's basically just declaring the IR version of ONNX protos we're using.  This wouldn't be dynamic, it's a static declaration of what ONNX version we're serializing to (ONNX 3).  Opset is similar, it's a slightly differently semantically versioned subset of operators within ONNX 3 that we're using.  So basically together we declare here that we're support ONNX 3, opset 8.  I'll update this to add some clarity tomorrow.

Reply:
I see. To me it's just not clear why these specific versions have been chosen and what will happen if either side gets updated. 
I'm on my phone, so sorry for being so Brief. Let me know if you would like be to elaborate my question

Reply:
No worries.  What you're asking is clear.  So the underlying library (onnx-tensorrt) moved to this version and opset soon after migrating to TRT5.  Almost immediately we began seeing serious correctness regressions with the old onnx version and opset, so updating them to a supported version seems like the path of least resistance to enable TRT5 support.  After changing the opset and updating the library the output is exactly as we'd expect (i.e. matching MXNets output).  So basically we want to update onnx-tensorrt to fix bugs and support new hardware, to do so we need to serialize onnx in exactly this format.